mk/artifact.mk

```diff
@@ -0,0 +1,68 @@
+ENABLE_PREBUILT ?= 1
+
+CC ?= gcc
+CROSS_COMPILE ?= riscv-none-elf-
+
+BIN_DIR := $(abspath $(OUT))
+
+TEST_SUITES += \
+	ansibench \
+	rv8-bench
+
+# "ieee754" needs F extension
+# "smolnes", "ticks" have inline assembly and only work in riscv
+TEST_BENCHES += \
+	captcha \
+	donut \
+	fcalc \
+	hamilton \
+	jit \
+	lena \
+	line \
+	maj2random \
+	mandelbrot \
+	nqueens \
+	nyancat \
+	pi \
+	puzzle \
+	qrcode \
+	richards \
+	rvsim \
+	spirograph \
+	uaes
+
+SHELL_HACK := $(shell mkdir -p $(BIN_DIR)/linux-x86-softfp $(BIN_DIR)/riscv32)
+
+ifeq ($(call has, PREBUILT), 1)
+  LATEST_RELEASE := $(shell wget -q https://api.github.com/repos/sysprog21/rv32emu-prebuilt/releases/latest -O- | grep -Po '(?<="tag_name": ").+(?=",)')
+else
+  CFLAGS := -m32 -mno-sse -mno-sse2 -msoft-float -O2 -L$(BIN_DIR)
```

Comment:
Add comments to emphasize the reason why linux-x86 target with soft floating point is built here.